modules/auth_oauth2/config.ts

```diff
@@ -0,0 +1,11 @@
+export interface Config {
```

Comment:
let's rename this to `customProviders` and add pre-configured blocks for `google`, `facebook`, etc.

see the repo i sent an while back for reference on pre-made configs

Reply:
Will do. This will require some reworking however, and IMO that functionality should be provided by a `google` or `facebook` module.

Reply:
ok cool let's get the interface stable then by just renaming this to "customProviders" and add presets later